hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -101,6 +101,7 @@ public synchronized int read(final byte[] b, final int off, final int len) throw
     int currentLen = len;
     int lastReadBytes;
     int totalReadBytes = 0;
+    incrementReadOps();
```

Comment:
this is input stream; presumably it's come in from somewhere else

Reply:
this is from the master PR(#1881)